src/Symfony/Component/Uid/AbstractUid.php

```diff
@@ -16,6 +16,10 @@
  */
 abstract class AbstractUid implements \JsonSerializable
 {
+    public const BASE58 = '[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{22}';
```

Comment:
I don't really know where to put those constants but they are very useful to be used in route requirements now that we have real annotations. I hope I don't have to use only the parameter version :pray:

Reply:
Yet they don't fit here. I need ideas :smile: